app/models/order.rb

```diff
@@ -31,4 +31,11 @@ def checkout(cart)
     end
   end
 
+  def self.by_status(status)
+    if Order.statuses.keys.include?(status)
```

Comment:
Can we clean this up some how with more AR-y methods? The hash chain seems a little hacky to me. @kheppenstall